device/bgp/bgpsrtesegment.yaml

```diff
@@ -3,47 +3,490 @@ components:
+            Optional SR-MPLS SID.
+          $ref: '#/components/schemas/BgpSrte.SrMplsSid'
+
+    BgpSrte.SegmentGTypeSubTlv:
+      description: >-
+        Type G: IPv6 Address, Interface ID for local and remote
+        pair with optional SID for SR MPLS.
+      type: object
+      required: [local_ipv6_node_address, remote_ipv6_node_address]
+      properties:
+        flags:
           description: >-
-            TBD
+            One octet of segment flags for V-Flag, A-Flag, S-Flag, B-Flag etc. as defined in
+            https://datatracker.ietf.org/doc/html/draft-ietf-idr-segment-routing-te-policy-13#section-2.4.4.2.12
           type: integer
+          minimum: 0
+          maximum: 255
           default: 0
+        local_interface_id:
+          description: >-
+            Local Interface ID: The Interface Index as defined in [RFC8664].
+          type: integer
+          minimum: 0
+          maximum: 2147483647
+          default: 0
+        local_ipv6_node_address:
+          description: >-
+            IPv6 address representing a node.
+          type: string
+          format: ipv6
+        remote_interface_id:
+          description: >-
+            Local Interface ID: The Interface Index as defined in [RFC8664].
+          type: integer
+          minimum: 0
+          maximum: 2147483647
+          default: 0
+        remote_ipv6_node_address:
+          description: >-
+            IPv6 address representing a node.
+          type: string
+          format: ipv6
+        sr_mpls_sid:
+          description: >-
+            Optional SR-MPLS SID.
+          $ref: '#/components/schemas/BgpSrte.SrMplsSid'
+
+    BgpSrte.SegmentHTypeSubTlv:
+      description: >-
+        Type H: IPv6 Local and Remote addresses with optional SID
+        for SR MPLS.
+      type: object
+      required: [local_ipv6_address, remote_ipv6_address]
+      properties:
+        flags:
+          description: >-
+            One octet of segment flags for V-Flag, A-Flag, S-Flag, B-Flag etc. as defined in
+            https://datatracker.ietf.org/doc/html/draft-ietf-idr-segment-routing-te-policy-13#section-2.4.4.2.12
+          type: integer
+          minimum: 0
+          maximum: 255
+          default: 0
+        local_ipv6_address:
+          description: >-
+            Local IPv6 Address.
+          type: string
+          format: ipv6
+        remote_ipv6_address:
+          description: >-
+            Remote IPv6 Address.
+          type: string
+          format: ipv6
+        sr_mpls_sid:
+          description: >-
+            Optional SR-MPLS SID.
+          $ref: '#/components/schemas/BgpSrte.SrMplsSid'
+
+    BgpSrte.SegmentITypeSubTlv:
+      description: >-
+        Type I: IPv6 Node Address with optional SRv6 SID.
+      type: object
+      required: [ipv6_node_address]
+      properties:
+        flags:
+          description: >-
+            One octet of segment flags for V-Flag, A-Flag, S-Flag, B-Flag etc. as defined in
```

Comment:
One octet bitmap for flags including V-Flag, A-Flag, S-Flag, B-Flag etc. as defined in